src/app/home/sidebar/categories/categories.component.html

```diff
@@ -24,7 +24,7 @@ <h4 *ngIf="categoryLevel.length > 0"> Catgeories</h4>
     <li *ngFor="let levels of categoryLevel; let index =i" (click)="emitSelection()">
       <a [routerLink]="['.']" class="list-group-item list-group-item-action" [queryParams]="{'q[name_cont]': levels.name, id: levels.id}">
        <span class="maninname">
-          {{levels.name}}
+          {{levels.c}}
```

Comment:
Why did you changed ? {{levels.name}} to {{level.c}}

Reply:
This done by mistake. Change reverted.